webapp/src/client/client.js

```diff
@@ -271,29 +257,62 @@ export default class Client {
             automaticLeaderAssignment: 'SameEnterprise',
         };
 
-        const response = await request.
-            post(url).
-            set('Authorization', 'Bearer ' + appAccessToken).
-            set('Accept', 'application/json').
-            send(data);
+        const response = await this.doPost(url, data, {Authorization: 'Bearer ' + appAccessToken, Accept: 'application/json'});
 
         return {
-            meetingId: response.body.onlineMeetingId,
-            meetingUrl: response.body.joinUrl,
+            meetingId: response.onlineMeetingId,
+            meetingUrl: response.joinUrl,
         };
     };
 
-    sendPost = async (url, body, headers = {}) => {
-        headers['X-Requested-With'] = 'XMLHttpRequest';
+    doGet = async (url, headers = {}, credentials) => {
+        const options = {
+            method: 'get',
+            headers,
+        };
+
+        const newOptions = Client4.getOptions(options);
```

Comment:
This should also be applied to `doGet` as well:

```suggestion
        let options = {
            method: 'get',
            headers,
        };
        
        if (url.includes('plugins/' + pluginID)) {
            options = Client4.getOptions(options);
        }
```